src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -72,10 +72,10 @@ public Stream<Event> fetchEventsInTimeRangeByServiceType(
   public Map<EventKey, Event> mapEventsInTimeRange(
       String orgId,
       String eventSource,
-      String eventType,
+      List<String> eventType,
```

Comment:
Should we make this a `Set`?  Seems like having duplicates in `eventType` wouldn't make any sense so we shouldn't even allow it in the first place.

Reply:
Done